libModule1/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id "java-library"
+    id "kotlin"
+}
+
+// or apply it per module through subprojects - root build.gradle
+apply from: "$rootDir/maven-publish.gradle"
+
+// specify per module - mostly needed due to different artifactIds, names, descriptions
+ext.mavenPublishProperties = [
+        group      : releaseConfig.group,
+        version    : releaseConfig.version,
+        artifactId   : "libModule1",
+        repository : [
+                url     : sonatype.url,
+                username: sonatype.username,
+                password: sonatype.password
+        ],
+        name       : "ExampleLib LibModule1",
+        description: "ExampleLib LibModule1 module",
+        url        : "https://github.com/infinum/android-libname",
+        scm        : [
+                connection: "https://github.com/infinum/android-libname.git",
+                url       : "https://github.com/infinum/android-libname"
+        ]
+]
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
```

Comment:
Is there a reason for 1.7? Actually, is there even a reason for 1.8? 😄

Reply:
Not sure how this happened, my bad, will change to 11 👍